packages/@aws-cdk/aws-codepipeline/test/cross-env.test.ts

```diff
@@ -119,7 +119,9 @@ describe.each(['legacy', 'modern'])('with %s synthesis', (synthesisStyle: string
 
         // THEN
         expect(supportStack).not.toHaveResource('AWS::KMS::Key');
-        expect(supportStack).toHaveResource('AWS::S3::Bucket');
+        expect(supportStack).toHaveResourceLike('AWS::S3::Bucket', {
+          PublicAccessBlockConfiguration: {},
```

Comment:
Hmm, not sure I understand this assertion. Shouldn't this be:

```suggestion
          PublicAccessBlockConfiguration: {
            BlockPublicAcls: true,
            BlockPublicPolicy: true,
            IgnorePublicAcls: true,
            RestrictPublicBuckets: true,
          },
```

?

Reply:
I was just looking for the block `PublicAccessBlockConfiguration ` presence. I will do your changes to also inspect the details of the block